libbeat/docs/shared-path-config.asciidoc

```diff
@@ -0,0 +1,96 @@
+//////////////////////////////////////////////////////////////////////////
+//// This content is shared by all Elastic Beats. Make sure you keep the
+//// descriptions here generic enough to work for all Beats that include
+//// this file. When using cross references, make sure that the cross
+//// references resolve correctly for any files that include this one.
+//// Use the appropriate variables defined in the index.asciidoc file to
+//// resolve Beat names: beatname_uc and beatname_lc.
+//// Use the following include to pull this content into a doc file:
+//// include::../../libbeat/docs/shared-path-config.asciidoc[]
+//// Make sure this content appears below a level 2 heading.
+//////////////////////////////////////////////////////////////////////////
+
+[[configuration-path]]
+=== Path
+
+The `path` section contains configuration options that define where the Beat
+is looking for certain types of files.
```

Comment:
The statement "certain types of files" is a bit too vague. Maybe be specific about the kinds of data that the files contain or how they are used.